hyperspy/io_plugins/fei.py

```diff
@@ -316,6 +316,97 @@ def get_xml_info_from_emi(emi_file):
     return objects[:-1]
 
 
+def get_calibration_from_position(position):
+    offset = position[0]
+    for i, x in enumerate(position):
+        if x != position[0]:
+            break
+    if i == len(position) - 1:
```

Comment:
Sorry that I forgot to comment on that. If the data is recorded by "uniform" rastering that cannot happen: if you plot the coordinates they should look like a sawtooth (first rastering axis) or staircase (second rastering axis). If it looks like a sawtooth the value repeats every "size" of the other axis. If no rastering occurs over the other axis, then it never repeats.